### ISSUE TITLE
No Admin Account Found

### ISSUE THREAD

**Issue by martin9115** [jguillod/couchdb-on-raspberry-pi#1]
Hi, create project. I have an issue:
After i run this command in CLI, (sudo -i -u couchdb /home/couchdb/bin/couchdb
) i've got next error

`No Admin Account Found, aborting startup.                  
  Please configure an admin account in your local.ini file. `

How to create new Account ?

**Comment by wintifrosch** [comment]
At the end of `/home/couchdb/etc/local.ini`, you must specify an admin password and uncomment the line.

**Comment by martin9115** [comment]
I dont have local.ini file

**Comment by wintifrosch** [comment]
@martin9115: in Step [4: First run and check](https://github.com/jguillod/couchdb-on-raspberry-pi#4-first-run-and-check) in the manual you were instructed to to edit the `/home/couchdb/etc/local.ini` file. At the end of this file, you will find a section labeled with [admins]: 
> `;To create an admin account uncomment the '[admins]' section below and add a`
> `; line in the format 'username = password'. When you next start CouchDB, it`
> `; will change the password to a hash (so that your passwords don't linger`
> `; around in plain-text files). You can add more admin accounts with more`
> `; 'username = password' lines. Don't forget to restart CouchDB after`
> `; changing this.`
> `[admins]`
> `admin = yoursecretpassword`

In older versions, there was a `#` character in front of a comment line, in recent versions you will find a `;`.  
You must add your own secret password on the last line, uncomment this line, save the file and restart CouchDB. 

@jguillod: please update step 4 in the manual. From CouchDB Docs [3.4 Authentication and Authorization](https://docs.couchdb.org/en/stable/config/auth.html#server-administrators)
> _Changed in version 3.0.0:_ CouchDB requires an admin account to start. If an admin account has not been created, CouchDB will print an error message and terminate.

**Comment by martin9115** [comment]
Thank you! I fix the problem and create database. Now i'm trying to sent sensor data via Node-red to  IBM Cloudant database . Can you help me?

**Comment by wintifrosch** [comment]
Yes, individual support is possible but may be liable to pay costs. DM [me](https://n3xt.ch/wpcms/#contact) or [jguillod](http://www.imed.ch/contact/index.html).

**Comment by martin9115** [comment]
Unfortunately  i can't afford it, but thank you for the help so far.
